catalog/migrate/iterator.go

```diff
@@ -81,46 +76,16 @@ func (it *Iterator) Value() *rocks.EntryRecord {
 }
 
 func (it *Iterator) Err() error {
-	return it.err
+	return it.rows.Err()
```

Comment:
```suggestion
	return it.err
```

Reply:
fixed - part of moving from using iterator error my our own err.